src/helpers/getPoolName.ts

```diff
@@ -0,0 +1,2 @@
+export const getPoolName = (assetA: string, assetB: string) =>
+  `${assetA}-${assetB}`;
```

Comment:
What if the helper accepted the array and did the whole magic internally? Then the accepted param would be a single array of asset codes.